src/sage/rings/finite_rings/element_base.pyx

```diff
@@ -268,6 +268,7 @@ cdef class FinitePolyExtElement(FiniteRingElement):
 
         TESTS::
 
+            sage: # needs sage.modules
```

Comment:
I can't see why this is needed. What command did you use to find this?

How can I find the distribution that installs this file `src/sage/rings/finite_rings/element_base.pyx`? Sorry I forgot...

Reply:
It would be nice if `--fixdoctests` finds the distribution for me...

Reply:
I am preparing a worktree with #35095, where perhaps I can run the command you used

Reply:
I failed in building sage with #35095...

Reply:
Many implementations of `random_element` need vectors.

Reply:
> How can I find the distribution that installs this file `src/sage/rings/finite_rings/element_base.pyx`?

In #35095, all files now have the `sage_setup: distribution` distribution header.
I'll update #36135 with the latest version of the tool that adds these headers.